irohad/ametsuchi/impl/mutable_storage_impl.cpp

```diff
@@ -79,6 +82,7 @@ namespace iroha {
         block_index_->index(*block);
 
         top_hash_ = block->hash();
+        top_height_ = block->height();
```

Comment:
Top ledger peers can be added here, and used in chainvalidator instead of peerquery.